roles/openshift_health_checker/test/etcd_traffic_test.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from openshift_checks.etcd_traffic import EtcdTraffic, OpenShiftCheckException
+
+
+@pytest.mark.parametrize('group_names,is_containerized,version,is_active', [
+    (['masters'], False, "3.5", True),
+    (['masters'], True, "3.6", False),
+    (['nodes'], False, "3.6", False),
+    (['nodes'], True, "3.4", False),
+    (['etcd'], False, "3.4", True),
+    (['etcd'], True, "3.5", True),
+    (['etcd'], False, "3.5", True),
+    (['masters', 'nodes'], False, "3.5", True),
+    (['masters', 'etcd'], False, "3.3", False),
+    ([], False, "3.4", False),
+    (['lb'], False, "3.6", False),
```

Comment:
Looks like we have more combinations here than needed to cover the logic.
E.g. the test case in this line produces is_active=False because the version is wrong and because the group_names doesn't contain the names we're interested in. We may be able to have less lines with the same effect.